app/Observers/PostObserver.php

```diff
@@ -98,6 +136,14 @@ public function saved(Post $post)
      */
     public function deleted(Post $post)
     {
+        $log = new EntityLog();
```

Comment:
I see 7 lines of code copy-pasted 3 times on this model. Please refactore to reduce the amounts of copy-pasted lines of code.